src/styles/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: $width;
+  height: 16px;
+  line-height: 16px;
+  padding: 16px 16px 0;
+
+  &--4 :nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__image {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+  }
+
+  &__text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+    padding-left: 17px;
+    color: $black;
```

Comment:
The variable `$black` is used here, but it is not defined in this file. Ensure that it is defined in a separate SCSS file or imported correctly.